cmp/options.go

```diff
@@ -202,7 +202,7 @@ type invalid struct{ core }
 
 func (invalid) filter(_ *state, _, _ reflect.Value, _ reflect.Type) applicableOption { return invalid{} }
 func (invalid) apply(s *state, _, _ reflect.Value) {
-	const help = "consider using AllowUnexported or cmpopts.IgnoreUnexported"
+	const help = "consider using Comparer. If you control the type, also consider AllowUnexported or cmpopts.IgnoreUnexported"
```

Comment:
`consider using a custom Comparer; if you control the implementation of type, you can also consider AllowUnexported or cmpopts.IgnoreUnexported`

Reply:
done